src/engine/enginebuffer.cpp

```diff
@@ -1196,15 +1196,24 @@ void EngineBuffer::postProcess(const int iBufferSize) {
 }
 
 void EngineBuffer::updateIndicators(double speed, int iBufferSize) {
-    VERIFY_OR_DEBUG_ASSERT(m_trackSampleRateOld && m_tempo_ratio_old) {
-        // no track loaded, function not called in this case
-        return;
+    if (!m_trackSampleRateOld) {
+       // this happens if Deck Passthrough is active but no track is loaded
```

Comment:
punctuation: "This happens if Deck Passthrough is active but no track is loaded.  We skip indicator updates."

Reply:
@daschuer Ping

Reply:
This should also be fixed.